tst/EnergyPlus/unit/FanCoilUnits.unit.cc

```diff
@@ -416,17 +416,8 @@ namespace EnergyPlus {
 
 		EXPECT_NEAR( QZnReq, QUnitOut, 5.0 );
 
-		DataGlobals::DoingSizing = false;
-
-		PlantLoop.deallocate();
-		ZoneSysEnergyDemand.deallocate();
-		FanCoil.deallocate();
-		Node.deallocate();
-		WaterCoil.deallocate();
-		ZoneEquipConfig.deallocate();
-		Zone.deallocate();
-		CoilNames.deallocate();
```

Comment:
Each unit test deallocated arrays which is done in clear_state. Deleted these as redundant.